.github/workflows/test2.yml

```diff
@@ -61,10 +61,7 @@ jobs:
           key: ${{ runner.os }}-${{ matrix.env.NPROC }}-nim-${{ hashFiles('.gitmodules') }}
 
       - name: Update dependencies
-        run: make -j${NPROC} V=1 update
-
-      - name: Build V2 binaries
```

Comment:
Shouldn't we always build the V2 binaries because some change in the waku/v2/ dir could break code which isn't part of the testing suite?